### PR TITLE
Hide points for appreciation

### PR DESCRIPTION
**Pull request by yatishTrootech**

### CODE REVIEW

File: feeds/models.py [hunk 1]
```diff
@@ -300,6 +300,28 @@ def feedback(self):
...
+        """
+        Returns points based on organization appreciation screen setting
+        If no transaction returns 0
+        If hide_points is True then points will be shown to the sender and receiver
+        """
+        transaction = self.transaction
+        if not transaction:
+            return 0
+
+        hide_points = user.organization.appreciation_screen_setting.get("hide_points") in (True, "true", "True")
```
Comment (rishireward) [inline feeds/models.py:313]: convert hide_points value to lowercase and then match
`str(hide_points_setting).lower() == 'true'`

File: feeds/models.py [hunk 2]
```diff
@@ -300,6 +300,28 @@ def feedback(self):
...
+        points = transaction.points
+        points = int(points) if points - int(points) == 0 else float(points)
```
Comment (rishireward) [inline feeds/models.py:315]: transaction.points will always be decimal as per the field definition. Why are we writing such complex logic here?
what's the purpose?

File: feeds/models.py [hunk 3]
```diff
@@ -300,6 +300,28 @@ def feedback(self):
...
+
+        if not hide_points:
+            return points
+
+        if user in (transaction.user, transaction.creator):
+            return points
```
Comment (rishireward) [inline feeds/models.py:321]: both if can be combined using OR

File: feeds/models.py (hunk 2 above)
Comment (yatishTrootech) [reply feeds/models.py:315]: it is existing code, 

Just not to break existing functionality that is why added same thing here

File: feeds/models.py (hunk 3 above)
Comment (yatishTrootech) [reply feeds/models.py]: done

**Review by rishireward — APPROVED** [review]

**Pull request merged**